torch/_inductor/cpp_builder.py

```diff
@@ -1545,3 +1547,65 @@ def build(self) -> Tuple[bytes, str]:
 
         _remove_dir(_build_tmp_dir)
         return status, self._target_file
+
+    def _remove_directory_from_file_path(self, file_path: str) -> str:
+        # Remove the directory part of file_path
+        return "${CMAKE_CURRENT_SOURCE_DIR}/" + Path(file_path).name
+
+    def save_compile_cmd_to_cmake(
+        self,
+        cmake_path: str,
+    ) -> None:
+        definitions = " ".join(self._build_option.get_definations())
+        contents = textwrap.dedent(
+            f"""
+            cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
```

Comment:
It does not feel like it needs any of 3.18 feature, so why put this constraint here?

Reply:
Copied from the main CMakeLists.txt in pytorch, but I agree it seems unnecessary here.

Reply:
Hmm, it actually causes the cuda test fail to compile. Not sure why, but I am keeping this.